tox.ini

```diff
@@ -0,0 +1,53 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+skipsdist = true
+envlist =
+    py35-sqlite
+    flake8
+    docs
+    isort
+
```

Comment:
@sergei-maertens Actually, I think I found a workaround to this. Adding two new environments, `py2` and `py3`, that use the default `python2` and `python3` binaries that exist on most systems, should accomplish what you are looking for without including both versions in the default environment list and without `skip_missing_interpreters = true`. PR is updated so you can see what I mean.